src/GovUk.Education.ExploreEducationStatistics.Admin/Migrations/ContentMigrations/20230224161839_EES3988KeyStatisticCleanUp.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using GovUk.Education.ExploreEducationStatistics.Common.Extensions;
+using Microsoft.EntityFrameworkCore.Migrations;
+using static GovUk.Education.ExploreEducationStatistics.Admin.Migrations.MigrationConstants;
+
+#nullable disable
+
+namespace GovUk.Education.ExploreEducationStatistics.Admin.Migrations.ContentMigrations
+{
+    public partial class EES3988KeyStatisticCleanUp : Migration
+    {
+        private const string MigrationId = "20230224161839";
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.DropColumn(
+                name: "ContentBlockIdTemp",
+                table: "KeyStatistics");
+
+            migrationBuilder.DropColumn(
+                name: "DataBlock_Summary",
+                table: "ContentBlock");
```

Comment:
How's the confidence level on charts not relying on anything to do with DataBlock.Summary? 🙂

Reply:
Pretty high. If it was being used, I would have expected to have seen something go red when I removed it and all associated view models etc. But nothing was using it except key stats. 

If there is a way for me to get caught out over this, I'm not sure what it is. I don't see how anything else could be accessing this column. If we were using javascript, I suspect this could have been a nightmare, but with it being typescript/c#, I think we're fairly safe.